### PR TITLE
Require `try` in teacher_dashboard_helper.

### PR DESCRIPTION
**Pull request by aoby**
https://github.com/code-dot-org/code-dot-org/pull/9959 introduced this [error](https://app.honeybadger.io/projects/34365/faults/30776776). It turns out `try` is not loaded in the Pegasus standalone context, but it is loaded when run via `bin/dashboard-server` on localhost so my earlier testing didn't turn this up. Running `bin/pegasus-server` alone and navigating to `/teacher-dashboard/landing` did repro it, and I verified this way that it is fixed.

### CODE REVIEW

File: pegasus/helpers/teacher_dashboard_helpers.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 def dashboard_admin?
-  dashboard_user.try{|u| u[:admin]}
+  return false unless dashboard_user
+  dashboard_user[:admin]
```
Comment (ashercodeorg) [inline pegasus/helpers/teacher_dashboard_helpers.rb]: Consider `dashboard_user && dashboard_user[:admin]`.

**Comment by ashercodeorg** [comment]
LGTM.

**Comment by wjordan** [comment]
You could also solve this by requiring the [#try extension](https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/object/try.rb) within the Pegasus-only code path directly:

```
require 'active_support/core_ext/object/try'
```

**Comment by aoby** [comment]
Thx. I'll use `require 'active_support/core_ext/object/try'` and verify it locally first...

**Pull request merged**